wicketd/src/update_tracker.rs

```diff
@@ -380,6 +388,26 @@ impl UpdateTrackerData {
         Self { artifact_store, sp_update_data: BTreeMap::new() }
     }
 
+    fn clear_update_state(
+        &mut self,
+        sp: SpIdentifier,
+    ) -> Result<(), ClearUpdateStateError> {
+        // Is an update currently running? If so, then reject the request.
+        //
+        // TODO: maybe add a way to force state to be cleared (this is
+        // dangerous!)
```

Comment:
Is "force state to be cleared" a different operation from "cancel a running update"? If we had cancelling a running update (which we should have eventually), would that remove the need to force a state clear?

Reply:
Ah yeah, it is the same thing. Thanks.